MSAL/src/MSALLogger.m

```diff
@@ -26,71 +26,45 @@
 //------------------------------------------------------------------------------
 
 #import "MSALLogger.h"
+#import "MSALGlobalConfig.h"
+#import "MSALLoggerConfig.h"
 
 @implementation MSALLogger
-{
-    MSALLogCallback _callback;
-}
 
 #pragma mark - Callback
 
 + (MSALLogger *)sharedLogger
 {
-    static dispatch_once_t once;
-    static MSALLogger *s_logger;
-    
-    dispatch_once(&once, ^{
-        s_logger = [MSALLogger new];
-        
-        [[MSIDLogger sharedLogger] setCallback:^(MSIDLogLevel level, NSString *message, BOOL containsPII) {
-            
-            if (s_logger->_callback)
-            {
-                s_logger->_callback((MSALLogLevel)level, message, containsPII);
-            }
-            
-        }];
-    });
-    
-    return s_logger;
+    return [self.class new];
```

Comment:
please make sure this returns a shared instance

Reply:
done